hack/deploy-karmada.sh

```diff
@@ -200,8 +200,10 @@ fi
 # deploy karmada apiserver
 TEMP_PATH_APISERVER=$(mktemp -d)
 trap '{ rm -rf ${TEMP_PATH_APISERVER}; }' EXIT
+KARMADA_APISERVER_VERSION=${KARMADA_APISERVER_VERSION:-"v1.27.11"}
```

Comment:
Is there a plan to bump this to v1.28?

Reply:
yes, i will  open a PR for it ASAP.

Reply:
Thanks. 
I'm asking this because I guess the default version of `karmadactl init`/`operator`/`chart` might need to update as well.

Reply:
I hope that the default version of karmada-apiserver will be consistent with the version of the dependent go module (WDYT), and will catch up to be consistent ASAP.

Reply:
That's a good idea if you mean the version of the `k8s.io/xxx` dependencies in go.mod file.

Reply:
k8s.io/client-go version  VS  kube-apiserver version